docker-compose.yml

```diff
@@ -156,10 +158,26 @@ services:
     depends_on:
       - db
     volumes:
+      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:ro
       - ./workspace-zen/services/events-service:/usr/src/app:Z
   maildev:
     image: djfarrelly/maildev
     ports:
       - '1080:80'
+  email:
+    image: node:carbon
+    command: yarn dev
+    working_dir: /usr/src/app
+    environment:
+      - EMAIL_DEFAULT_FROM='The CoderDojo Team <info@coderdojo.org>'
+      - MICROSERVICE_PORT=3000
+      - TRANSPORT=smtp
+      - SMTP_HOST=maildev
+      - TEMPLATES_DIR=/usr/src/app/node_modules/cp-translations/email-templates
+      - SMTP_SECURE=false
+      - SMTP_PORT=25
+    volumes:
+      - ./workspace-zen/services/email-service:/usr/src/app:Z
```

Comment:
Shouldn't this be `cp-email-service`? That's the repo that actually runs the service, right?

Reply:
shoudl also be added to the setup script :smiling_imp:

Reply:
Just gotta look up at the review comment 😛 

> Add the new service to setup repos bash script.

Reply:
nah :stuck_out_tongue_closed_eyes: